services/dy-static-file-server/docker/custom/boot.sh

```diff
@@ -8,25 +8,17 @@ echo   User    : "$(id "$(whoami)")"
 echo   Workdir : "$(pwd)"
 echo   Env      : "$(env)"
 
+echo "/workdir content"
+ls -lah
 
-if [ -n "${SIMCORE_NODE_BASEPATH+set}" ]
-then
-    echo
-    echo moving website to "${NGINX_SERVER_ROOT}${SIMCORE_NODE_BASEPATH}"...
-    echo
-
-    mkdir -p "${NGINX_SERVER_ROOT}${SIMCORE_NODE_BASEPATH}"
-    mv "${NGINX_SERVER_ROOT}"/*.txt "${NGINX_SERVER_ROOT}${SIMCORE_NODE_BASEPATH}"
+echo "ensure some random data is created in  in /workdir/generated-data content"
+/venv/bin/python ensure_random_workdir_data.py
 
-    echo "Nginx will serve from ${NGINX_SERVER_ROOT}${SIMCORE_NODE_BASEPATH}"
-    
-else
-    echo "Nginx will serve from ${NGINX_SERVER_ROOT}, nothing to do"
-fi
+echo "/workdir/generated-data content"
+ls -lah /workdir/generated-data/
 
-# ensure some random data is created in /workdir
-/venv/bin/python ensure_random_workdir_data.py
+echo "starting background inputs->ouputs transformation"
+/venv/bin/python inputs_to_outputs.py &
```

Comment:
if you add the /venv/bin/python to the PATH in the dockerfile (also see the article linked above), you might be able to simplify these calls.